### PR TITLE
Move custom-style out of dom-module

### PR DESCRIPTION
**Pull request by cdata**
/cc @morethanreal

### CODE REVIEW

**Comment by notwaldorf** [comment]
I think `toggle-button` will need this too :) lgtm

File: paper-checkbox.html [hunk 1]
```diff
@@ -43,18 +43,17 @@
...
-      --paper-checkbox-unchecked-ink-color: var(--primary-text-color);
-
-      --paper-checkbox-checked-color: var(--default-primary-color);
-      --paper-checkbox-checked-ink-color: var(--default-primary-color);
-    }
-  </style>
+    --paper-checkbox-checked-color: var(--default-primary-color);
+    --paper-checkbox-checked-ink-color: var(--default-primary-color);
+  }
+</style>
```
Comment (morethanreal) [inline paper-checkbox.html]: Can the defaults go in the element style instead?

Comment (cdata) [inline paper-checkbox.html]: Yesterday we decided to do the element defaults in separate PRs since it's not a broken API. I can go either way.

Comment (morethanreal) [inline paper-checkbox.html]: Ok. I don't know what's been decided. Just commenting, since there's an open PR already?

**Pull request merged**